deployments/docker/.env.example

```
@@ -10,3 +10,14 @@
 KAFKA_PARTITIONS_HIGH=6
 KAFKA_PARTITIONS_NORMAL=8
 KAFKA_PARTITIONS_LOW=4
+
+# chromedp pool — worker:Chrome 1:1 매핑 (이슈 #230)
+# WORKER_COUNT 와 REMOTE_URLS 길이가 일치해야 함 (LoadFetcherChromedpPool 가 fail-fast 검증).
+# docker-compose 의 chrome-N 컨테이너 수도 함께 조정 필요.
+FETCHER_CHROMEDP_POOL_ENABLED=true
+FETCHER_CHROMEDP_WORKER_COUNT=2
+FETCHER_CHROMEDP_SEMAPHORE_CAPACITY=1
+# 호스트에서 실행 시 (docker-compose 외부 — 호스트 포트 9222/9223 사용):
+FETCHER_CHROMEDP_REMOTE_URLS=ws://localhost:9222,ws://localhost:9223
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FETCHER_CHROMEDP_REMOTE_URLS key should go before the FETCHER_CHROMEDP_SEMAPHORE_CAPACITY key

(UnorderedKey)